docs/transpile/rest-api-transpilation.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Transpilation using REST APIs
+description: How to transpile quantum circuits using REST APIs.
+---
+
+# Transpilation using REST APIs
+
+The circuit provided will need to be mapped to instructions fitting to the hardware basis instructions of the QPU, which is a process known as Transpilation.
```

Comment:
This introduction feels a bit too abrupt, can you rewrite this to introduce what the page will be covering before jumping into this note about the circuit?